src/component/mxgraph/MarkerConfigurator.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2020 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { MxGraphFactoryService } from '../../service/MxGraphFactoryService';
+import { mxgraph } from 'ts-mxgraph';
+
+export enum MarkerConstant {
+  ARROW_DASH = 'dash',
+}
+
+export default class MarkerConfigurator {
+  private mxMarker: typeof mxgraph.mxMarker = MxGraphFactoryService.getMxGraphProperty('mxMarker');
+
+  public configureMarkers(): void {
+    // Registers and defines the custom markers
+    this.registerArrowDashMarker();
+  }
+
+  private registerArrowDashMarker(): void {
+    // This implementation is adapted from the draw.io BPMN 'dash' marker
+    // https://github.com/jgraph/drawio/blob/f539f1ff362e76127dcc7e68b5a9d83dd7d4965c/src/main/webapp/js/mxgraph/Shapes.js#L2796
+
+    const createMarker =
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      (c: mxgraph.mxXmlCanvas2D, shape: mxgraph.mxShape, type: any, pe: any, unitX: number, unitY: number, size: number, source: mxgraph.mxCell, sw: any): any => {
```

Comment:
:question: could we use more explicit argument names? I guess the name are taken from mxGraph functions, but this is a pain to understand the arguments purpose

Reply:
I don't know what the arguments are, and their types :( 
https://jgraph.github.io/mxgraph/docs/js-api/files/shape/mxMarker-js.html

Reply:
Agree, not very explicit, I had a look at https://github.com/jgraph/mxgraph2/blob/mxgraph-4_1_1/javascript/src/js/shape/mxMarker.js#L65 and didn't get enough insights to conclude
At least, the type of `pe` could be set to `mxPoint` (https://github.com/jgraph/mxgraph2/blob/mxgraph-4_1_1/javascript/src/js/util/mxPoint.js)

Reply:
and `sw` seems to relate to `strokewidth` (see https://github.com/jgraph/mxgraph2/blob/mxgraph-4_1_1/javascript/src/js/shape/mxMarker.js#L57)

Reply:
for type, this is to be a `string`
- used as key in a map to register marker via `mxMarker.addMarker`
- mxMarker code do assertion on type value against strings: https://github.com/jgraph/mxgraph2/blob/mxgraph-4_1_1/javascript/src/js/shape/mxMarker.js#L68